troubleshooting/known_issues/cray_console_node_in_CLBO_state.md

```diff
@@ -0,0 +1,85 @@
+# `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state after fresh CSM install
+
+## Issue Description
+
+After installing CSM 1.6, `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state. This happens because of
```

Comment:
Is this problem actually intermittent? As in, does it recover on its own once it happens? Or is it something that once it happens, it will keep happening until the workaround is applied?